### ISSUE TITLE
Create ModuleApp and MaterialModuleApp widgets

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#18934]
We should have variants of WidgetsApp and MaterialApp for add-to-app scenarios. They would avoid doing things like Title, but still do things like Navigator and Theme.

**Comment by Hixie** [comment]
I guess it's not technically "add-to-app" if you don't have an app already, true, even if you're adding Flutter to an OEM part. I really meant "not-entirely-Flutter" scenarios.

**Comment by matthew-carroll** [comment]
@xster I wanted to make sure this ticket was on your radar.

**Comment by xster** [comment]
Not sure how important this is from the description. Seems like a P3.

**Comment by matthew-carroll** [comment]
Yeah, probably low priority, but also this ticket could easily be lost in the shuffle if we don't track it somewhere in the effort.

**Comment by xster** [comment]
I added it to phase 3 non fullscreen views

**Comment by dnfield** [comment]
I'm going to move this to stretch goals until we have a customer asking for this.
